pkg/granted/registry/registry_cmd.go

```diff
@@ -0,0 +1,37 @@
+package registry
+
+import (
+	"github.com/common-fate/clio"
+	grantedConfig "github.com/common-fate/granted/pkg/config"
+	"github.com/urfave/cli/v2"
+)
+
+var ProfileRegistry = cli.Command{
+	Name:        "registry",
+	Description: "Add git repository of AWS profiles which will be synced to ~/.aws/config file",
+	Subcommands: []*cli.Command{&AddCommand, &SyncCommand, &RemoveCommand},
+	Action: func(c *cli.Context) error {
+
+		gConf, err := grantedConfig.Load()
+		if err != nil {
+			return err
+		}
+
+		if len(gConf.ProfileRegistryURLS) <= 0 {
+			clio.Info("There are no registry subscribed to granted yet.\n Use 'granted registry add <your_repo>' to configure subscription.\n")
```

Comment:
Reading over the prompts here, should we switch the wording from: `registry` to `repository` since we are referring to git repos?

```
There are no repositories subscribed to granted yet.\n Use 'granted registry add <your_repo>' to configure subscription.
```